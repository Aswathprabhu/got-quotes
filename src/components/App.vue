<template>
  <div id="app">
    <Card />
  </div>
</template>

<script>
import 'isomorphic-fetch';
import Card from './Card';

export default {
  name: 'App',
  components: {
    Card,
  },
};
</script>

<style>
html,
body {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  height: 100%;
}
#app {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
}
</style>
