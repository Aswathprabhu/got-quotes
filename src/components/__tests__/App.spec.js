import App from '../App.vue';
import { render } from '@testing-library/vue';

describe('App Page - Initial Render', () => {
  it('Initial render', () => {
    const { container } = render(App);
    expect(container).toMatchSnapshot();
  });
});
