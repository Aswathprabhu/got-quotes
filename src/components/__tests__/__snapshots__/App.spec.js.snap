// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Page - Initial Render Initial render 1`] = `
<div>
  <div
    id="app"
  >
    <div
      class="card"
    >
      <div
        class="loader-container"
      >
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
      </div>
    </div>
     
    <div
      class="card"
    >
      <div
        class="loader-container"
      >
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
      </div>
    </div>
     
    <div
      class="card"
    >
      <div
        class="loader-container"
      >
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
      </div>
    </div>
     
    <div
      class="card"
    >
      <div
        class="loader-container"
      >
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
         
        <div
          class="loader sm"
        />
         
        <div
          class="loader"
        />
      </div>
    </div>
  </div>
</div>
`;
